client/src/components/Chat/MessageInput.jsx

```diff
@@ -31,11 +34,13 @@ const MessageInput = ({
 			<form className="flex justify-center items-center mt-[40px]" onSubmit={handleSubmit}>
 				<div className="w-full flex items-center justify-between bg-secondary rounded-l-md max-h-[150px] relative">
 					<textarea
-						placeholder="Send a Message....."
+						placeholder={focusTextArea ? 'Press Ctrl + Enter to send message':'Type a message...'}
```

Comment:
Just the text, no need for a state

Reply:
@Dun-sin  addressed all the comments , thanks for the review 😃